WrapImage/nifti_wrapper.py

```diff
@@ -1,11 +1,51 @@
 import argparse
 import json
 import os
+from pathlib import Path
+import subprocess
 import nibabel as nib
 from src.wrappers.OsipiBase import OsipiBase
 import numpy as np
 from tqdm import tqdm
 
+def dicom_to_niix(vol_dir: Path):
+    """
+    For converting DICOM images to a (compresssed) 4d nifti image
+    """
+
+    try:
+        res = subprocess.run(
+            [
+                "dcm2niix",
+                "-f", "%s_%p", # dcm2niix attempts to provide a sensible file naming scheme
+                "-o", vol_dir,
+                "-z", "y", #specifying compressed nii.gz file
+                # https://www.nitrc.org/plugins/mwiki/index.php/dcm2nii:MainPage
+                # for further configuration for general usage see page above
+                vol_dir         
```

Comment:
Is this outputting into the same folder as the input? It should probably be a temporary folder or perhaps sub folder to prevent mixing of the image types.